<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNewGame</class>
 <widget class="QDialog" name="DlgNewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a new game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupName">
     <property name="title">
      <string>Enter a game name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="editName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRace">
     <property name="title">
      <string>Pick a race</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonProtoss">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Pictures.qrc">
          <normaloff>:/Images/racelogos/Protoss.png</normaloff>:/Images/racelogos/Protoss.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonsRace</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTerran">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Pictures.qrc">
          <normaloff>:/Images/racelogos/TerranGray.png</normaloff>:/Images/racelogos/TerranGray.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonsRace</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonZerg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Pictures.qrc">
          <normaloff>:/Images/racelogos/ZergGray.png</normaloff>:/Images/racelogos/ZergGray.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonsRace</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayoutButtons">
     <item>
      <spacer name="hspacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>buttonProtoss</tabstop>
  <tabstop>buttonTerran</tabstop>
  <tabstop>buttonZerg</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="Pictures.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonsRace"/>
 </buttongroups>
</ui>
